packages/agoric-cli/lib/deploy.js

```diff
@@ -21,7 +21,7 @@ import { getAccessToken } from './open';
 
 const RETRY_DELAY_MS = 1000;
 
-const PATH_SEP_RE = new RegExp(`${path.sep}`, 'g');
```

Comment:
ocap grumble: module constant computed from ambient authority (`path.sep`).

cf. #2160

Reply:
There's obviously a lot further to go before there's no ambient authority in the `agoric-cli`.